tutorial/lesson-3/README.md

```diff
@@ -67,8 +67,8 @@ is explained on the page [Why Not ERB](erb.md).
 
 ## Stubs
 
-Four files will be created in the `lib` directory, `entry_flow.rb`,
-`entry_view.rb`, `entry_update.rb`, and `entry_list.rb`.
+For this lesson, four files will be created in the `lib` directory,
```

Comment:
If the user should create and copy the content from the text below.
I would phrase it 

> For this lesson , you need to create four files in the `lib` directory"

To avoid the confusion of thinking these are files that were created automatically by `wabur` in advance.

Reply:
ok